sagemaker-triton/resnet50/jit_trace/Triton_CPU_JIT_MME.ipynb

```diff
@@ -0,0 +1,1534 @@
+{
```

Comment:
<p>Capitalize the first letter of a senetence</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3739/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>